src/Browser/GoogleChrome/VersionResolver.php

```diff
@@ -37,9 +37,24 @@ public function from(OperatingSystem $operatingSystem, string $path): Version
         }
 
         if ($operatingSystem->equals(OperatingSystem::WINDOWS())) {
-            return $this->getVersionFromCommandLine(
-                sprintf('wmic datafile where name="%s" get Version /value', $path)
-            );
+            $possibleCommands = [
+                'reg query HKLM\Software\Google\Update\Clients\{8A69D345-D564-463c-AFF1-A69D9E530F96} /v pv /reg:32 2> NUL',
+                'reg query HKLM\Software\Google\Update\Clients\{8237E44A-0054-442C-B6B6-EA0509993955} /v pv /reg:32 2> NUL',
+                'reg query HKLM\Software\Google\Update\Clients\{401C381F-E0DE-4B85-8BD8-3F3F14FBDA57} /v pv /reg:32 2> NUL',
+                'reg query HKCU\Software\Google\Update\Clients\{8A69D345-D564-463c-AFF1-A69D9E530F96} /v pv /reg:32 2> NUL',
+                'reg query HKCU\Software\Google\Update\Clients\{8237E44A-0054-442C-B6B6-EA0509993955} /v pv /reg:32 2> NUL',
+                'reg query HKCU\Software\Google\Update\Clients\{401C381F-E0DE-4B85-8BD8-3F3F14FBDA57} /v pv /reg:32 2> NUL',
+                'reg query HKCU\Software\Google\Update\Clients\{4ea16ac7-fd5a-47c3-875b-dbf4a2008c20} /v pv /reg:32 2> NUL',
+            ];
```

Comment:
Yes it is maybe not the solution for all cases but at least it works for finding on Windows version of Chrome installed "normally".
I have no idea which version it will then provide if two different version of Chrome is installed (probably the one of the last run Chrome)